calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,168 @@
+
+For the 2d SVD-based method, since radar detections lack a z component, we reduce the problem to 2d by setting the z component of lidar detections to zero. We then estimate the rigid transformation using the SVD-based method provided by PCL, which leverages SVD to find the optimal rotation component and then calculates the translation component based on the rotation.
+
+The yaw-only rotation method, on the other hand, calculates the average yaw angle difference of all pairs and estimates the transformation, considering only yaw rotation, between the sensors. Generally, the 2d SVD-based method is preferred when valid; otherwise, the yaw-only rotation method is used as the calibration output.
+
+It's also important to note that in the near future, the calibrator will be updated to support radar that includes elevation angles and provides different transformation algorithms.
+
+### Diagram
+
+Below, you can see how the algorithm is implemented in the `marker_radar_lidar_calibrator` package.
+
+![marker_radar_lidar_calibrator](../docs/images/marker_radar_lidar_calibrator/marker_radar_lidar_calibrator.jpg)
+
+## ROS Interfaces
+
+### Input
+
+| Name                     | Type                            | Description                                |
+| ------------------------ | ------------------------------- | ------------------------------------------ |
+| `input_lidar_pointcloud` | `sensor_msgs::msg::PointCloud2` | Topic of lidar pointcloud for calibration. |
+| `input_radar_msg`        | `radar_msgs::msg::RadarTracks`  | Topic of radar objects for calibration.    |
+
+### Output
+
+| Name                          | Type                                   | Description                                     |
+| ----------------------------- | -------------------------------------- | ----------------------------------------------- |
+| `lidar_background_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Background pointcloud from lidar.               |
+| `lidar_foreground_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Foreground pointcloud from lidar.               |
+| `lidar_colored_clusters`      | `sensor_msgs::msg::PointCloud2`        | Colored pointcloud clusters from lidar.         |
+| `lidar_detection_markers`     | `visualization_msgs::msg::MarkerArray` | Lidar detections.                               |
+| `radar_background_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Background pointcloud from radar.               |
+| `radar_foreground_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Foreground pointcloud from radar.               |
+| `radar_detection_markers`     | `visualization_msgs::msg::MarkerArray` | Radar detections.                               |
+| `matches_markers`             | `visualization_msgs::msg::MarkerArray` | Matched lidar and radar detection.              |
+| `tracking_markers`            | `visualization_msgs::msg::MarkerArray` | Tracks of reflectors.                           |
```

Comment:
Reflectors' tracks

I was put it as an automatic suggestion, but that would probably make pre commit fail